src/project/internal/projectmigrator.cpp

```diff
@@ -167,7 +164,7 @@ Ret ProjectMigrator::migrateProject(engraving::EngravingProjectPtr project, cons
         m_resetStyleSettings = false;
     }
 
-    if (ok && opt.isApplyAutomaticPlacement) {
+    if (ok && score->mscVersion() < 300) {
```

Comment:
FWIW, I'm not so sure this is really what I'd expect either.  But, if that's actually all the checkbox was doing before - controlling whether manual adjustments were reset or not - it's harmless to keep the box.  It's just terribly named, then - if the point of the box is the reset positions to the (new/better) defaults, that's what it should say.

Still, since it's trivial to do Ctrl+A / Ctrl+R after loading the score, I'd just as soon see the checkbox go away, *and* see this block of code away go away with it, so older scores simply come through as is, and the user can then decide if they want to reset positions.  After all, if we do reset them here, it will be impossible to get them back (unless "undo" immediately on import accomplishes that, but still, that's not obvious).

Reply:
Is the point not that this resets manual adjustments made in files from pre-MU3, i.e. before the autoplace era, whicih are now likely to be meaningless? Seems better to me to present the user with something (hopefully) sensible rather than something messed-up which they need to take an explicit step (even if "trivial") to fix. In any case the checkbox should go.

Reply:
I assume that is the point, yes.  But, we have several years of experience here already to tell us, it's really a toss-up as to whether resetting all positions when importing MU2 scores into MU3 produces better results than not.  Some of those manual adjustments might be counterproductive, others might be unnecessary but harmless, others still actually important for the layout effect desired.

But, it's a bigger jump from MU2 to MU4, obviously; so that much more likely those old manual adjustments are not good.  Therefore, maybe not a *terrible* idea to just toss them.  I was just surprised to realize that this is what this checkbox does, and suddenly not so sure getting rid of it is warranted.